packages/app-builder-lib/templates/nsis/include/extractAppPackage.nsh

```diff
@@ -113,13 +113,21 @@
       # Try copying a few times before asking for a user action.
       Goto RetryExtract7za
     ${else}
-      MessageBox MB_RETRYCANCEL|MB_ICONEXCLAMATION "$(appCannotBeClosed)" /SD IDCANCEL IDRETRY RetryExtract7za
+      MessageBox MB_ABORTRETRYIGNORE|MB_ICONEXCLAMATION "$(appCannotBeClosed)" /SD IDABORT IDABORT AbortExtract7za IDRETRY RetryExtract7za
```

Comment:
Do you the silent default (`/SD`) should be ABORT? I think it is the safest option, but just wanted to make sure it doesn't go unnoticed.